### ISSUE TITLE
Can't enable COPR repo on ARM devices (aarch64)

### ISSUE THREAD

**Issue by fenik** [gsauthof/dracut-sshd#41]
I recently installed Fedora Server 33 on a Raspberry Pi and noticed that the repo can't be enabled on it. The reason seems to be that DNF doesn't check the x86_64 subrepo for noarch packages and thus disallows enabling the repository stating that there are no builds available.

A workaround for those having the same issue would be as follows: Download the repo file manually and edit the `base_url` attribute to exclude the `$basearch` variable.

E. g. 
`baseurl=https://download.copr.fedorainfracloud.org/results/gsauthof/dracut-sshd/fedora-$releasever-$basearch/`
should be changed to
`baseurl=https://download.copr.fedorainfracloud.org/results/gsauthof/dracut-sshd/fedora-$releasever-x86_64/`

**Comment by gsauthof** [comment]
Thank you for reporting this.

This is caused by the configuration of that Copr repository where one can select a subset of the many available distribution architectures.

The aarch64 variant wasn't consistently enabled, e.g. it was enabled for f32 but not for f33.

So I've just added:

```
fedora-rawhide-aarch64
fedora-33-aarch64
```

Since that repository auto-follows Fedora's branching, fedora-34-aarch64 and following will be created automatically.

It's true that it would be nice if Copr would make noarch packages available to all architectures, by default - since one binary package could be used for all.

I triggered a build for aarch64, thus installing it via copr should now also work on f33 aarch64.
